src/test/rgw/test_ratelimit_unittest.cc

```diff
@@ -0,0 +1,191 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab ft=cpp
+
+#include <gtest/gtest.h>
+#include "rgw/rgw_qos.h"
+
+
+using namespace std::chrono_literals;
+
+TEST(RGWRateLimit, shouldratelimit1)
+{
+  // info.enabled = false, so no limit
```

Comment:
these test comments are helpful, but i think they'd be more helpful as the test names. for example,
```
TEST(RGWRateLimit, op_limit_not_enabled)
TEST(RGWRateLimit, reject_op_over_limit)
TEST(RGWRateLimit, accept_op_after_refill)
TEST(RGWRateLimit, reject_bw_over_limit)
```
etc